### PR TITLE
feat: remove yat banner on ETH sends

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - to be followed-up by more yat cleanups

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/6182

None

> What protocols, transaction types or contract interactions might be affected by this PR?

## Testing

- There is no Yat callout on ETH sends

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

- This diff

Screenshot 2024-03-28 at 10 37 30

- Develop

image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6597** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6597?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/Modals/Send/views/Address.tsx [hunk 1]
```diff
@@ -129,15 +121,12 @@ export const Address = () => {
           <AddressInput
             rules={addressInputRules}
             enableQr={true}
-            placeholder={translate(
-              isYatSupported ? 'modals.send.addressInput' : 'modals.send.tokenAddress',
-            )}
+            placeholder={translate('modals.send.addressInput')}
```
Comment (kaladinlight) [inline src/components/Modals/Send/views/Address.tsx:124]: this is the translation that we want. no longer need `addressInput` (even though that is the better translation name if we want to update)
```suggestion
            placeholder={translate('modals.send.tokenAddress')}
```

Comment (kaladinlight) [reply src/components/Modals/Send/views/Address.tsx:124]: seems like we would still support ENS though, so maybe we want to update the translation just to remove `YAT`

Comment (gomesalexandre) [reply src/components/Modals/Send/views/Address.tsx:124]: While this does look sane from a "just remove yat logic standpoint", this was previously wrong IMO, i.e:
https://github.com/shapeshift/web/blob/df13ac5de88d90ef7c29edd86998dca7d1e53a14/src/assets/translations/en/main.json#L1015
vs. https://github.com/shapeshift/web/blob/df13ac5de88d90ef7c29edd86998dca7d1e53a14/src/assets/translations/en/main.json#L1016

Comment (kaladinlight) [reply src/components/Modals/Send/views/Address.tsx:124]: Can follow up when removing yat validation entirely I guess since this is just banner work.

**Review by kaladinlight — APPROVED** [review]

File: src/components/Modals/Send/views/Address.tsx (hunk 1 above)
Comment (gomesalexandre) [reply src/components/Modals/Send/views/Address.tsx:124]: Though reading into this more, I believe we should probably keep the `ethChainId` checks and render `addressInput` if true (we only support ENS resolution on ETH mainnet), else `tokenAddress` 👀

Comment (kaladinlight) [reply src/components/Modals/Send/views/Address.tsx:124]: Yeah, good catch-ception to find actual change we want haha

Comment (gomesalexandre) [reply src/components/Modals/Send/views/Address.tsx]: ec01f00e5d3d415b1e6d30aafd9d5339b3d3f735

**Pull request merged**